sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -649,6 +649,52 @@ def _repr_html_(self):
 class_inst
 """
 
+code_mime_plain = """
+class mime_plain():
```

Comment:
It's a nitpick, but I'm finding it ugly that there is a class that is not CamelCass.